mapping/requirements_officejs.json

```diff
@@ -2874,6 +2874,17 @@
 							}],
 							"availability": "GA"
 						},
+						{
+							"name": "WordApiDesktop",
+							"apiVersion": "1.1",
+							"supportedProductVersions": [{
+								"from": {
+									"build": "16.85.0.0",
+									"version": null
+								}
+							}],
+							"availability": "GA"
```

Comment:
just a placeholder, will wait for July fork build number later. Don't merge so far.